README.md

```diff
@@ -49,6 +49,7 @@ It looks for the first occurrence in HTML and sets its value to a field.
 * `Float`
 * `Double`
 * `Date`
+* `BigDecimal`
```

Comment:
Also it is needed to add new type to javadocs [here](https://github.com/DroidsOnRoids/jspoon/blob/master/jspoon/src/main/java/pl/droidsonroids/jspoon/annotation/Selector.java#L16)